kano_feedback/DataSender.py

```diff
@@ -500,9 +510,7 @@ def send_question_response(question_id, answer, interactive=True):
 
         if retry.run():
             # Try again until they say no
-            return send_question_response(question_id=question_id,
-                                          answer=answer,
-                                          interactive=interactive)
+            return send_question_response((question_id, answer), interactive=interactive)
```

Comment:
I think that this will need to be given a list of tuples, i.e.

```python
return send_question_response([(question_id, answer)], interactive=interactive)
```